manifests/f/StefanFreischlad/Filius/2.1.0/StefanFreischlad.Filius.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: StefanFreischlad.Filius
+PackageVersion: 2.1.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://www.lernsoftware-filius.de/downloads/Setup/Filius-Setup_with-JRE-2.1.0.exe
+  InstallerSha256: 2B03D98407595A4CEF276E4AED9641762B82504E430025F3F1C533A0533F8757
+ManifestType: installer
+ManifestVersion: 1.2.0 

```

```suggestion
ManifestVersion: 1.2.0
```
nit: Trailing whitespace